test/cypress/integration/transactions/transaction-search.cy.js

```diff
@@ -334,6 +334,64 @@ describe('Transactions List', () => {
       assertTransactionRow(1, filteredByMultipleFieldsTransactions[1].reference, `/account/${gatewayAccountExternalId}/transactions/payment-transaction-id2`,
         'test@example.org', '–£15.00', 'Visa', 'Refund submitted')
     })
+
+    it('should be display error message when searching with from date later than to date', () => {
+      cy.task('setupStubs', [
+        ...sharedStubs(),
+        transactionsStubs.getLedgerTransactionsSuccess({ gatewayAccountId, transactions: unfilteredTransactions }),
+        transactionsStubs.getLedgerTransactionsSuccess({
+          gatewayAccountId,
+          transactions: filteredByDatesTransactions,
+          filters: {
+            from_date: '2018-05-03T00:00:00.000Z',
+            to_date: '2018-05-03T00:00:01.000Z'
+          }
+        })
+      ])
+      cy.visit(transactionsUrl)
+
+      // 1. Filtering FROM
+      // Ensure both the date/time pickers aren't showing
+      cy.get('.datepicker').should('not.exist')
+      cy.get('.ui-timepicker-wrapper').should('not.exist')
+
+      // Fill in a from date
+      cy.get('#fromDate').type('03/5/2018')
+
+      // Ensure only the datepicker is showing
+      cy.get('.datepicker').should('be.visible')
+      cy.get('.ui-timepicker-wrapper').should('not.exist')
+
+      // Fill in a from time
```

Comment:
I think we should get rid of some of these comments.
As we can tell from the code what it's doing.

Reply:
Agreed.
Comments are in so many tests and many files, I guess it will extend the scope of the ticket even further.
So, I have remove the comments in the affected test scenario.